packages/jest-cli/src/cli/args.js

```diff
@@ -357,6 +357,15 @@ export const options = {
     description: "A preset that is used as a base for Jest's configuration.",
     type: 'string',
   },
+  pretransformFiles: {
+    description:
+      'Find all test dependencies before the run, transform their ' +
+      'source and write transformed source cache files. By doing it before the ' +
+      'actual test run Jest can make sure that files are not transformed twice ' +
+      'as well as take advantage of doing it in parallel. This is useful when ' +
+      'running Jest on CI on cold cache ',
```

Comment:
Does this mean that if this option is provided, we don't have to recommend `runInBand`? http://facebook.github.io/jest/docs/en/troubleshooting.html#tests-are-extremely-slow-on-docker-and-or-continuous-integration-ci-server

Reply:
i'm not sure if it'll help with this specific case. I only saw benefits when running jest on a lot of cores (24-40), but i'd be curious to see if it changes the way CI runs :)